src/staticdata_utils.c

```diff
@@ -621,6 +621,7 @@ JL_DLLEXPORT uint8_t jl_cache_flags(void)
     flags |= (jl_options.opt_level & 3) << 4;
     // NOTES:
     // In contrast to check-bounds, inline has no "observable effect"
```

Comment:
Btw @vchuravy, what does this mean?

Reply:
Uhm... That I have not used `--inline` ever. 

We should probably still split the cache since if you accidentially compile a pkgimg with `--inline` it will poison the cache due to https://github.com/JuliaLang/julia/blob/de73c26fbff61d07a38c9653525b530a56630831/base/util.jl#L140 & https://github.com/JuliaLang/julia/blob/de73c26fbff61d07a38c9653525b530a56630831/base/loading.jl#L1947

My original reasoning was that whether we inline or not is not an observable effect, since it is a pure optimization.

Reply:
>  it will poison the cache due to

Yeah, I was thinking if you run with `--inline=no` you poison it with very slow code.